make/hotspot/lib/CompileJvm.gmk

```diff
@@ -135,6 +135,10 @@ else ifeq ($(call isTargetCpu, x86_64), true)
   JVM_EXCLUDE_PATTERNS += x86_32
 endif
 
+ifeq ($(call isTargetCpu, riscv64), true)
+  JVM_CFLAGS += $(RVV_CFLAGS)
```

Comment:
The new flag should be added to the JVM_CFLAGS at configure time. No need to export it to spec.gmk and do it at runtime.

Reply:
The flag is different for the build JVM and the JVM.
If I just set in configure time it seem like both OPENJDK_BUILD_JVM_CFLAGS and JVM_CFLAGS have the flag.
Which is incorrect, as only OPENJDK_BUILD_JVM_CFLAGS in this case should have it.

Am I doing something wrong? I mirrored SVE_FLAGS which do this...

Reply:
`SVE_CFLAGS` is used to compile a separate library, libsleef. They do not have an already prepared set of flags to use, as Hotspot do. (Hotspot is the only library which has this, since it is magnitutes more complex than other libraries.)

Reply:
Having different flags for the build and the target JVM is a bit more tricky, but not impossible. You just need to "hålla tungan rätt i mun". :-) 

In `FLAGS_SETUP_CFLAGS_CPU_DEP`, you need to add something like:
```
  if test "x$FLAGS_CPU" = xriscv64; then
    $1_DEFINES_CPU_JVM="$1_DEFINES_CPU_JVM $2RVV_DEFINES"
  fi
```

Reply:
Thanks, I'll test it.

Reply:
It is written untested so it might need some tweaking. My suggestion also incorporated (silently) the recommendation to rename  $2RVV_FLAG to $2RVV_DEFINES. :)

Reply:
I did a simpler version. Let me know :)
The tongue may have slipped :)